components/vc/pdu/src/HW/HW_gpio_componentSpecific.c

```diff
@@ -52,3 +52,637 @@ const HW_GPIO_S HW_GPIO_pinmux[HW_GPIO_COUNT] = {
         .resetState = HW_GPIO_PINRESET,
     },
 };
+
+
+
+/*----------------------*/
+
+[HW_GPIO_BMS2_EN] = {
+    .port = GPIOB,
+    .pin = GPIO_PIN_8,
+    .mode = GPIO_MODE_OUTPUT_PP,
+    .speed = GPIO_SPEED_FREQ_LOW,
+    .pull = GPIO_NOPULL,
+    .resetState = HW_GPIO_PINRESET,
+},
+[HW_GPIO_ACCUM_EN] = {
+    .port = GPIOB,
+    .pin = GPIO_PIN_8,
+    .mode = GPIO_MODE_OUTPUT_PP,
+    .speed = GPIO_SPEED_FREQ_LOW,
+    .pull = GPIO_NOPULL,
+    .resetState = HW_GPIO_PINRESET,
+},
+[HW_GPIO_LP2_LATCH] = {
+    .port = GPIOB,
+    .pin = GPIO_PIN_8,
+    .mode = GPIO_MODE_OUTPUT_PP,
+    .speed = GPIO_SPEED_FREQ_LOW,
+    .pull = GPIO_NOPULL,
+    .resetState = HW_GPIO_PINRESET,
+},
+[HW_GPIO_BSPD_MEM] = {
+    .port = GPIOB,
+    .pin = GPIO_PIN_8,
+    .mode = GPIO_MODE_OUTPUT_PP,
+    .speed = GPIO_SPEED_FREQ_LOW,
+    .pull = GPIO_NOPULL,
+    .resetState = HW_GPIO_PINRESET,
+},
+[HW_GPIO_SPI_CS_SD] = {
+    .port = GPIOB,
+    .pin = GPIO_PIN_8,
+    .mode = GPIO_MODE_OUTPUT_PP,
+    .speed = GPIO_SPEED_FREQ_LOW,
+    .pull = GPIO_NOPULL,
+    .resetState = HW_GPIO_PINRESET,

```

Comment:
If it is setup as pushpull, and it initializes reset, you will be initializing and then commanding theslave spi device because its active low. Change the mode to OUTPUT_OD and the resetstate will be good